tests/Browser/Back/PasswordResetTest.php

```diff
@@ -40,11 +48,15 @@ public function it_can_reset_passwords()
             $browser
                 ->visit($passwordResetUrl)
                 ->assertSee('WACHTWOORD WIJZIGEN')
-                ->type('password', 'secret')
-                ->type('password_confirmation', 'secret')
+                ->type('password', $newPassword)
+                ->type('password_confirmation', $newPassword)
                 ->press('Wachtwoord instellen')
                 ->assertPathIs('/blender')
                 ->assertSee(__('passwords.reset'));
         });
+
+        $user = User::where('email', $user->email)->first()->makeVisible('password');
+
+        $this->assertTrue(Hash::check($newPassword, $user->password));
```

Comment:
I don't think `makeVisible` is necessary here. Line 58 could be omitted, and the password can be fetched using `$user->fresh()->password` (I think)